compiler/rustc_infer/src/infer/opaque_types.rs

```diff
@@ -372,7 +370,7 @@ impl<'tcx> InferCtxt<'tcx> {
     }
 
     #[instrument(skip(self), level = "trace", ret)]
-    pub fn opaque_type_origin(&self, def_id: LocalDefId, span: Span) -> Option<OpaqueTyOrigin> {
+    pub fn opaque_ty_origin(&self, def_id: LocalDefId, span: Span) -> Option<OpaqueTyOrigin> {
```

Comment:
Right below this function, we do already have `opaque_ty_origin_unchecked`, which could live on TyCtxt as a method.